pkg/bpf/map.go

```diff
@@ -234,11 +286,24 @@ func (m *Map) OpenOrCreate() (bool, error) {
 		os.Remove(m.path)
 	}
 
+reopen:
```

Comment:
my previous comment is regarding this `goto`, the `fd` open on the next line doesn't seem to be properly closed (I think)